config/packages/doctrine.yaml

```
@@ -9,6 +9,6 @@ doctrine:
     dbal:
         driver: 'pdo_mysql'
         server_version: '5.7'
-        charset: UTF8
+        charset: UTF8mb4
```

```suggestion
        charset: utf8mb4
```

That looks like an official spelling, according to MySQL documentation.